common/path.cc

```diff
@@ -989,9 +988,8 @@ absl::StatusOr<std::string> GetSymlinkTarget(const std::string& link_path) {
   std::filesystem::path symlink_target =
       std::filesystem::read_symlink(link_path_u8, error_code);
   if (error_code) {
```

Comment:
Same here, you could get rid of this condition.

Reply:
In that case, it doesn't return an absl::Status, so it doesn't work here.